LEGO1/omni/src/video/mxstillpresenter.cpp

```diff
@@ -55,7 +55,7 @@ void MxStillPresenter::CreateBitmap()
 	m_frameBitmap = new MxBitmap;
 	m_frameBitmap->ImportBitmapInfo(m_bitmapInfo);
 
-	delete m_bitmapInfo;
+	delete[] m_bitmapInfo;
```

Comment:
Can you contribute this to original decomp. thank you!

Reply:
and add a cast to MxU8* before delete. also there's at least one other incorrect delete in the file. thanks

Reply:
Here or in the other PR?

Reply:
in the decomp pr